### ISSUE TITLE
Import Error in PandasAIReader[Bug]: 

### ISSUE THREAD

**Issue by georgenavarro1802** [run-llama/llama_index#14878]
### Bug Description

I would like to report here an **error or problem** regarding the **PandasAIReader** reader, which uses the **pandasai** library behind it. Well, it turns out that pandasai in its latest versions **has eliminated the PandasAI** class and then, there is an error when PandasAIReader tries to import the PandasAI class from the pandasai module.

this line of code:
`from pandasai import PandasAI`
inside the constructor of the PandasAIReader class

The way I have found to work for now is to use an older version of Pandasai, but the correct thing to do in llamaindex is to update this import block, so that it supports the same old Pandasai as the new one where this PandasAI class is no longer present.

I put a link from Pandasai's github where it shows that they have eliminated the PandasAI class:
[pandas ai github commit](https://github.com/Sinaptik-AI/pandas-ai/commit/8345b2208db7ca959a81e54a9d719b40113f50ff)

### Version

0.10.56

### Steps to Reproduce

1. call: loader = PandasAIReader(llm=llm)

and check that the class PandasAIReader(BaseReader) has the following block of code in its __init__ file:

```
try:
    from pandasai import PandasAI
    from pandasai.llm.openai import OpenAI
except ImportError:
    raise ImportError("Please install pandasai to use this reader.")
```

the line causing the error is: from pandasai import PandasAI because PandasAI class does not longer exist in pandasai library

**Comment by logan-markewich** [comment]
This would be a pretty easy PR if you are interested

**Comment by georgenavarro1802** [comment]
Hi @logan-markewich  it's not a pretty easy PR. 
pandasai made so many changes and deprecated all the PandasAI class and related entites thru the entire codebase. 
The only classes that are allowed now are: Agent, SmartDataframe and SmartDataLake.
I am trying to adapt or readjust the PandasAI class locally, to use any of those classes, looking for a way to create nodes / documents in order to have a correct structure to be passed to the llamaindex vectorstore or any other store, but I have not yet found a satisfactory result. I have to redefine several methods internally

**Comment by aaryan200** [comment]
Hello,
I've tried to fix this error in #14915 . But one of the test named Linting is failing. Afaik, it's a static check that runs on the code.

The issue is probably with the argument `is_conversational_answer` which is neither used by [pandasai.PandasAI.run](https://github.com/Sinaptik-AI/pandas-ai/commit/8345b2208db7ca959a81e54a9d719b40113f50ff#diff-62b472d83558337a7726ff6758a8c431481784a74add715c96e2fed8bedbbd63L159) method nor by [pandasai.SmartPDFReader.chat](https://github.com/Sinaptik-AI/pandas-ai/blob/main/pandasai/smart_dataframe/__init__.py#L110) method.

Is it okay to remove that argument from `load_data` method of `PandasAIReader`?

If you have any other suggestion, please let me know.

**Comment by aaryan200** [comment]
Thank you @logan-markewich !
